src/test/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/OffenderPendingDeletionRepositoryTest.java

```diff
@@ -32,30 +30,6 @@
     @Autowired
     private OffenderPendingDeletionRepository repository;
 
-    @Test
```

Comment:
I've removed these tests as this code will be deleted once the new delete data from nomis service is complete from @hyderkhanmoj

Reply:
Good to remove as it's replaced in this PR: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerdeletionservice/repository/jpa/DeceasedOffenderPendingDeletionRepositoryTest.kt

Reply:
@hyderkhanmoj I've approved that PR